### PR TITLE
Xml parsing

### PR DESCRIPTION
**Pull request by javakid0826**
This branch adds the functionality to fetch and parse the xml data from the specified data source

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -17,11 +19,20 @@ app.use('/js/lib', express.static(path.join(__dirname, 'node_modules/popper.js/d
 app.use('/js/lib', express.static(path.join(__dirname, 'node_modules/bootstrap/dist/js')));
 app.use('/js/lib', express.static(path.join(__dirname, 'node_modules/jquery/dist')));
 
-app.get("/data/*", (req, res) => {
-	console.log(req._parsedUrl.path);
-	res.header("Access-Control-Allow-Origin", "*");
-	request({url:"https://dd.weather.gc.ca/citypage_weather/xml/"}, (error, response, body) => {
-		return(body);
+// app.get("/data/*", (req, res) => {
```
Comment (schmidtvl) [inline app.js]: Is this commented out code no longer needed? If so, let's remove it. Best practices for adding new code into master is to only keep what is needed.

File: js/script.js [hunk 2]
```diff
@@ -1,10 +1,71 @@
 console.log("Hello World");
 
+const tagRgx = /(<(?!\/)([^>\s]*)([^>]*)(?!\/)>)/g;
```
Comment (schmidtvl) [inline js/script.js]: This variable is not being used in this file so it's probably safe to delete it (but see comment below.)

File: js/script.js [hunk 3]
```diff
@@ -1,10 +1,71 @@
...
 }
 
-xhr.open("GET", "/data/", true);
-xhr.send();
\ No newline at end of file
+xhr.open("GET", "/data/siteList.xml");
+xhr.send();
+
+
+// DEPRECATED: I tried to create my own XML parser but it ultimately ended up being a dead end
```
Comment (schmidtvl) [inline js/script.js]: Similar comment here, if this code is not being used let's remove it.

However, if you want to keep this code to work on it in the future, maybe the thing to do would be to extract it into a separate file in the `js` directory. You could give this file a name that's descriptive for what the code is intended to do.

File: package.json [hunk 4]
```diff
@@ -1,27 +1,18 @@
...
-  "contributors": [
-    "Yashar Fakhari (https://github.com/YasharF)"
-  ],
+  "description": "A test to try and fetch and display data from a source",
   "scripts": {
-    "postinstall": "npm rebuild node-sass",
-    "start": "node app.js",
-    "test": "nyc mocha --timeout=10000 --exit",
-    "lint": "eslint \"**/*.js\""
+    "start": "npx nodemon app.js"
```
Comment (schmidtvl) [inline package.json]: Great!

File: js/script.js [hunk 5]
```diff
@@ -1,10 +1,71 @@
...
+// 			parseXML(sub, xml[tagName]);	
+// 		} else {
+// 			xml["innerContent"] = sub;
+// 		}
+
+// 		workingString = workingString.substring(closeInd + close[0].length);
+// 	}
+
+// 	return xml;
+// }
```
Comment (schmidtvl) [inline js/script.js]:
```suggestion
// }

```

File: js/script.js [hunk 6]
```diff
@@ -1,10 +1,71 @@
...
 
-xhr.open("GET", "/data/", true);
-xhr.send();
\ No newline at end of file
+xhr.open("GET", "/data/siteList.xml");
+xhr.send();
+
+
+// DEPRECATED: I tried to create my own XML parser but it ultimately ended up being a dead end
+// const parseXML = (raw, xml = {}) => {
```
Comment (javakid0826) [inline js/script.js]:
```suggestion
```

File: js/script.js (hunk 5 above)
Comment (javakid0826) [inline js/script.js]:
```suggestion
```

File: js/script.js [hunk 7]
```diff
@@ -1,10 +1,18 @@
 console.log("Hello World");
 
+const tagRgx = /(<(?!\/)([^>\s]*)([^>]*)(?!\/)>)/g;
```
Comment (javakid0826) [inline js/script.js]:
```suggestion
```

**Pull request merged**